index.bs

```diff
@@ -727,14 +724,12 @@ if (relative_path === null) {
 </div>
 
 <div algorithm>
-The <dfn method for=FileSystemDirectoryHandle>resolve(|possibleDescendant|)</dfn> method,
-when invoked, must return the result of [=entry/resolving=]
-|possibleDescendant|'s [=FileSystemHandle/entry=] relative to [=this=]'s [=FileSystemHandle/entry=].
+The <dfn method for=FileSystemDirectoryHandle>resolve(|possibleDescendant|)</dfn> method
+must return the result of [=entry/resolving=]|possibleDescendant|'s
+[=FileSystemHandle/entry=] relative to [=this=]'s [=FileSystemHandle/entry=].
```

Comment:
Just a heads up that this is a slightly change that doesn't exactly follow the pattern